packages/modelserver-client/src/model-server-client-v2.spec.ts

```diff
@@ -362,23 +368,23 @@ describe('tests for ModelServerClientV2', () => {
             });
         });
 
-        it('ping ', done => {
+        it('getAll', done => {
             const model1: Model = {
-                modelUri: 'path/to/model1',
+                modeluri: 'path/to/model1',
                 content: {
                     name: 'coffee'
                 }
             };
             const model2: Model = {
-                modelUri: 'path/to/model2',
+                modeluri: 'path/to/model2',
```

Comment:
why did you change `modelUri` to `modeluri`?

Reply:
We basically used `modeluri` wherever possible, there were some leftovers especially in `v2` with used `modelUri`, I aligned everything to lowercase (to match the style of the actual used parameter).
There is one place left, where we use camel case, this is due to the data structure the model server sends, I guess this could be aligned at some point on the Java side to avoid any conflicts.